letsdrill.rb

```diff
@@ -1,13 +1,30 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+  case integer
+    when 80..89
+      return "B"
+    when 70..79
+      return "C"
+    when 60..69
+      return "D"
+    else
+      if integer > 90
+        return "A"
+      else
+        return "F"
+      end
+  end
 end
 
 def shortest_string(array)
+  longest = array[0]
+  array.each do |v| 
+    if longest.nil? then longest = v
+    elsif (v.length < longest.length) then longest = v 
+    end
```

Comment:
Here's a few suggestions.

1. I would name it `shortest` since that is what you're looking for.

2. I think you could probably condense it down to just a single if statement. You don't really need to do a nil check for each element in the array.

3. I would [take a look](https://apidock.com/ruby/Enumerable/min_by) at the `min_by` method.